spec/requests/duplicate_logs_controller_spec.rb

```diff
@@ -66,7 +66,6 @@
             it "displays check your answers for each log with correct questions" do
               expect(page).to have_content("Q5 - Tenancy start date", count: 3)
               expect(page).to have_content("Q7 - Tenant code", count: 3)
-              expect(page).to have_content("Q12 - Postcode", count: 3)
```

Comment:
Why does this no longer show up?

Reply:
It's a question that was pre-2023, I've removed the page and question